python_modules/libraries/dagster-dg-cli/dagster_dg_cli/api_layer/schemas/run_event.py

```diff
@@ -16,6 +16,24 @@ class RunEventLevel(str, Enum):
     DEBUG = "DEBUG"
 
 
+class ErrorInfo(BaseModel):
```

Comment:
This should be `DgApiErrorInfo`. (Also the one below should be `DgApiRunEvent`, that might be my fault and you don't need to change it in this PR)